airgun/widgets.py

```diff
@@ -1321,6 +1321,45 @@ class EditableEntryMultiCheckbox(EditableEntry):
     edit_field = CheckboxGroup(locator='.//form')
 
 
+class TextInputsGroup(GenericLocatorWidget):
+    """
+    A set of text inputs
+    """
+    ITEMS_LOCATOR = './/label'
```

Comment:
that sounds too ambiguous. What are `ITEMS`? Is it TEXTINPUT` group?

Reply:
Items are field lablels. TEXTINPUT_LOCATOR are text inputs found by their label.

But I've found another issue: the locator has changed in 6.6. I'll change the locator so it works for 6.6 and will fix it again for 6.5 during cherrypick, is that ok?

Reply:
that is intended procedure. So can we rename ITEMS to FIELD_LABELS or whatever you like?